.gitignore

```diff
@@ -1,4 +1,5 @@
 /user-config.yml
+lib/
```

Comment:
I'm not sure if it's just a convention, but trailing and leading slashes aren't required in .gitignores. `lib` matches the `lib/` directory.